sssom/cli.py

```diff
@@ -153,16 +182,30 @@ def parse(
     prefix_map_mode: str,
     clean_prefixes: bool,
     output: TextIO,
+    mapping_predicate_filter: tuple,
 ):
     """Parse a file in one of the supported formats (such as obographs) into an SSSOM TSV file."""
-    parse_file(
-        input_path=input,
-        output=output,
-        input_format=input_format,
-        metadata_path=metadata,
-        prefix_map_mode=prefix_map_mode,
-        clean_prefixes=clean_prefixes,
-    )
+    # Get list of predicates of interest.
+    if mapping_predicate_filter:
+        mapping_predicates = _get_list_of_predicate_iri(mapping_predicate_filter)
+        parse_file(
+            input_path=input,
+            output=output,
+            input_format=input_format,
+            metadata_path=metadata,
+            prefix_map_mode=prefix_map_mode,
+            clean_prefixes=clean_prefixes,
+            mapping_predicates=mapping_predicates,
+        )
+    else:
```

Comment:
I did this as well in parsers.py but I really hate it... Is there no better way? Using args *? Maybe there is a better design?

Reply:
In this case here, in any case, let the if-else be in parse_file()? If mapping_set is passed as`None`, it should work fine.

Reply:
Maybe something like this, but even here I am unsure: https://www.geeksforgeeks.org/python-star-or-asterisk-operator/#:~:text=Here%20single%20asterisk(%20*%20)%20is,and%20variable%2Dlength%20argument%20list.

Reply:
I think it'll work with the if-else statement. There's no need for the `*`. I've changed it.